package.json

```diff
@@ -69,7 +69,7 @@
     "passport-jwt": "^4.0.0",
     "randomstring": "^1.1.5",
     "raven": "^2.6.4",
-    "redis": "^2.8.0",
+    "redis": "^3.0.2",
```

Comment:
https://github.com/NodeRedis/node-redis/blob/v3.0.2/CHANGELOG.md#v300---09-feb-2020

We're not affected by the breaking changes and it mentions some performance benefits which would be nice